silver/migrations/0036_auto_20170514_1627.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.db.models import Q
+
+import silver.models.documents.pdf
+from silver.models import BillingDocumentBase
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('silver', '0035_auto_20170206_0941'),
+    ]
+
+    def move_pdf_from_documents_to_model(apps, schema_editor):
+        db_alias = schema_editor.connection.alias
+
+        Invoice = apps.get_model('silver', 'Invoice')
+        Proforma = apps.get_model('silver', 'Proforma')
+        PDF = apps.get_model('silver', 'PDF')
+
+        for invoice in Invoice.objects.using(db_alias).filter(
+            ~Q(state=BillingDocumentBase.STATES.DRAFT)
+        ):
+            pdf_object = PDF.objects.using(db_alias).create(
+                upload_path=invoice.get_pdf_upload_path()
```

Comment:
Why don't you create the pdf with the `pdf_file` already set up?
Do you need to call `save()` ?